openassessment/xblock/static/js/philu/lib/helpers.js

```diff
@@ -0,0 +1,82 @@
+(function () {
+
+    var oa_table_builder_init_helpers = {
+        oa_response_init: function (event) {
+            if (!$(event.target).is('li')) {
+                return;
+            }
+
+            event.preventDefault();
+            $(this).find('li.submission__answer__part[data-prompt-type="html-control"]').each(function (index, oa_block_li) {
```

Comment:
let's make constants for all dom element selectors

Reply:
done